### PR TITLE
Add push queue stats to Hermes

### PR DESCRIPTION
**Pull request by cdonati**
/stats/cluster/push_queues now reports the number of messages in each queue.

### CODE REVIEW

File: Hermes/appscale/hermes/stats/producers/rabbitmq_stats.py [hunk 1]
```diff
@@ -75,3 +93,44 @@ def get_current():
...
+      raise APICallFailed('Call to {} failed: {}'.format(url, error))
+
+    try:
+      queues_info = json.loads(response.body)
+    except ValueError:
+      raise APICallFailed('Invalid response from '
+                          '{}: {}'.format(url, response.body))
+
+    queue_stats = [
+      PushQueueStats(name=queue['name'], messages=queue['messages'])
```
Comment (jeanleonov) [inline Hermes/appscale/hermes/stats/producers/rabbitmq_stats.py]: Is "messages" number of messages?

File: Hermes/appscale/hermes/stats/stats_app.py [hunk 2]
```diff
@@ -170,6 +188,7 @@ def get_cluster_stats_api_routes(is_master):
     '/stats/cluster/processes': cluster_processes_stats_handler,
     '/stats/cluster/proxies': cluster_proxies_stats_handler,
     '/stats/cluster/rabbitmq': cluster_rabbitmq_stats_handler,
+    '/stats/cluster/push_queues': cluster_push_queue_stats_handler,
```
Comment (jeanleonov) [inline Hermes/appscale/hermes/stats/stats_app.py]: Stub handler is not initialized
```
Traceback (most recent call last):
  File "/usr/local/bin/appscale-hermes", line 9, in <module>
    load_entry_point('appscale-hermes==0.0.6', 'console_scripts', 'appscale-hermes')()
  File "/usr/local/lib/python2.7/dist-packages/appscale/hermes/__init__.py", line 252, in main
    + stats_app.get_cluster_stats_api_routes(is_master),
  File "/usr/local/lib/python2.7/dist-packages/appscale/hermes/stats/stats_app.py", line 191, in get_cluster_stats_api_routes
    '/stats/cluster/push_queues': cluster_push_queue_stats_handler,
UnboundLocalError: local variable 'cluster_push_queue_stats_handler' referenced before assignment
```

File: Hermes/appscale/hermes/stats/producers/rabbitmq_stats.py (hunk 1 above)
Comment (cdonati) [reply Hermes/appscale/hermes/stats/producers/rabbitmq_stats.py]: Yes. It's the number of messages waiting to be consumed.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/4562/

**Pull request merged**